README.md

```diff
@@ -32,7 +32,7 @@ composer require taxjar/taxjar-php
 ```json
 {
   "require": {
-    "taxjar/taxjar-php": "^1.8"
+    "taxjar/taxjar-php": "^1.10"
```

Comment:
Looks like this change was forgotten in the last couple releases. Updating now, better late than never. ☺️